### PR TITLE
Add US Army Loadouts

### PR DESCRIPTION
**Pull request by jokoho48**

### CODE REVIEW

**Comment by jokoho48** [comment]
@GenericSubject please assign Milestone
26 this weekend
28 next weekend(or later)

**Comment by GenericSubject** [comment]
done

File: lib/gear/fn_crate.sqf [hunk 1]
```diff
@@ -68,13 +68,12 @@ _fnc_medical = {
...
         ["ACE_medicalSupplyCrate_advanced", 5] call JK_fnc_createCrate;
     };
-
 };
 
 _fnc_ammo = {
     [
         "Box_NATO_Wps_F",
-        10,
+        15,
```
Comment (jokoho48) [inline lib/gear/fn_crate.sqf:76]: Change Spawn Time?

File: lib/gear/loadouts/special/fn_jetPilot.sqf [hunk 2]
```diff
@@ -19,10 +19,10 @@ JK_useProfileGoggles = 0;        //If set to 1, goggles from your profile will b
 JK_goggles = "G_Aviator";
 
 //weapons - primary weapon - (string)
-JK_primaryweapon = ["rhs_weap_m16a4", "rhs_weap_m16a4_grip", "rhs_weap_m16a4_carryhandle", "rhs_weap_m16a4_carryhandle_grip_pmag", "rhs_weap_m16a4_carryhandle_grip", "rhs_weap_m16a4_carryhandle_pmag", "rhs_weap_m4", "rhs_weap_m4_grip2", "rhs_weap_m4_grip", "rhs_weap_m4_carryhandle", "rhs_weap_m4_carryhandle_pmag"];
+JK_primaryweapon = ["hgun_PDW2000_F"];
 
 //primary weapon items - (array)
-JK_itemsPrimaryweapon = [["rhsusf_acc_SFMB556", "rhsusf_acc_SF3P556", ""]];
+JK_itemsPrimaryweapon = [["optic_Aco_smg"]];
```
Comment (jokoho48) [inline lib/gear/loadouts/special/fn_jetPilot.sqf]: use ["optic"] instand of [["optic"]] for a Optic without randomisation

File: lib/gear/loadouts/special/fn_pilot.sqf [hunk 3]
```diff
@@ -19,10 +19,10 @@ JK_useProfileGoggles = 0;        //If set to 1, goggles from your profile will b
 JK_goggles = "G_Aviator";
 
 //weapons - primary weapon - (string)
-JK_primaryweapon = ["rhs_weap_m16a4", "rhs_weap_m16a4_grip", "rhs_weap_m16a4_carryhandle", "rhs_weap_m16a4_carryhandle_grip_pmag", "rhs_weap_m16a4_carryhandle_grip", "rhs_weap_m16a4_carryhandle_pmag", "rhs_weap_m4", "rhs_weap_m4_grip2", "rhs_weap_m4_grip", "rhs_weap_m4_carryhandle", "rhs_weap_m4_carryhandle_pmag"];
+JK_primaryweapon = ["hgun_PDW2000_F"];
 
 //primary weapon items - (array)
-JK_itemsPrimaryweapon = [["rhsusf_acc_SFMB556", "rhsusf_acc_SF3P556", ""]];
+JK_itemsPrimaryweapon = [["optic_Aco_smg"]];
```
Comment (jokoho48) [inline lib/gear/loadouts/special/fn_pilot.sqf]: same as above

File: lib/gear/fn_crate.sqf (hunk 1 above)
Comment (GenericSubject) [inline lib/gear/fn_crate.sqf:76]: should be allright

File: lib/gear/loadouts/special/fn_jetPilot.sqf (hunk 2 above)
Comment (GenericSubject) [inline lib/gear/loadouts/special/fn_jetPilot.sqf]: changed it

**Comment by GenericSubject** [comment]
has https://github.com/jokoho48/AWPublicMission/issues/30 issue

**Pull request merged**